lib/reporters/base.js

```diff
@@ -219,6 +219,11 @@ exports.list = function(failures) {
     stack = stack.replace(/^/gm, '  ');
 
     console.log(fmt, (i + 1), test.fullTitle(), msg, stack);
+
+    // [hack] Handle waterline's nested `originalError`
+    if (err.originalError && err.originalError instanceof Error) {
```

Comment:
(I mean, it's always `instanceof Error` coming from Waterline, but I can't say for sure that no other module uses the same pattern with a non-`Error`. But even then, it would be fine.)